### ISSUE TITLE
Accessibility - Dataverse - <li> elements

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#94]
### Homepage: https://data.qdr.syr.edu/ 

- [x] Each of the options under the Sort menu (Name A-Z, Name Z-A, Newest, Oldest) 
![image](https://user-images.githubusercontent.com/89411909/134701630-0db9fa1b-73d0-42db-8743-d2b05ce30461.png)

- [x] Each of the options under the Data Download Menu
![image](https://user-images.githubusercontent.com/89411909/135911423-42ba3589-b2dd-400b-8ee9-6e2c40b1bb34.png)

### (Sample Page): Data for: Monster or Hero? Far-right Responses to Anders Behring Breivik and the 22 July Terrorist Attacks (https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6MBCJ8M)
- [x] Cite Data Project Drop Down Menu Items (all 3)
![image](https://user-images.githubusercontent.com/89411909/138486482-11faf258-752d-4c00-973d-d1ebb253aa59.png)

### Sample Page): M4 Eight Divided by One-Fourth Elementary Mathematics Task: https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6BCQ6DM
- [ ] Each of the M4 Files for Pre-Service Teacher Educators
![image](https://user-images.githubusercontent.com/89411909/140096094-846fb5bc-f269-4f7d-9cce-7553a7ac075d.png)

**Comment by qqmyers** [comment]
The menus should be fixed - mostly related to using the `role='menu'` . The last example is a problem with the PrimeFaces tree widget and is not yet fixed.
